Python-Home-Challenges/recurring word.py

```diff
@@ -1,10 +1,6 @@
 
 where_file = input('Enter the file route: ')
 my_file = open(where_file,"r")
-print("Your file text is:")
-for x in my_file:
-  print(x)
-print("---------------")
 words_in_text = dict()
 my_file = open(where_file,"r")
```

Comment:
you dont need to open file again

Reply:
fix.
strange, i was sure that i removed it.